Easydict/Feature/Service/OneAPI/OneAPIService.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  OneAPIService.swift
+//  Easydict
+//
+//  Created by phlpsong on 2024/2/16.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import Alamofire
+import CryptoKit
+import Defaults
+import Foundation
+
+@objc(EZOneAPIService)
```

Comment:
Please rename all `OneAPIService` to `CustomOpenAIService`

Reply:
Already changed in prev commits.